src/config-site/scratlantis.cmake

```diff
@@ -1,14 +1,14 @@
-#/Users/miller86/visit/third_party/3.3.0/cmake/3.24.3/darwin-x86_64/bin/cmake
```

Comment:
should scratlantis.cmake be in this PR?

Reply:
Thanks for doing this by the way. This is a good band-aid to have while we try to figure out what is going wrong under the hood.

Reply:
> should scratlantis.cmake be in this PR?

Well, I have to update it before doing any more work on `develop` so I just thought I'd let it go along for the ride. Ok?